addon/mirage/factory.js

```diff
@@ -1,3 +1,21 @@
+import Ember from 'ember';
+
+function generateValue(type, key) {
```

Comment:
Unmatched '{'.<br>'generateValue' is defined but never used.<br>'key' is defined but never used.